native-jfr-demo/README.md

```diff
@@ -0,0 +1,73 @@
+# Native Executables with JFR Example
+
+This example demonstrates how JDK Flight Recorder (JFR) as a tool for collecting diagnostic and profiling data running Java application, built into a JVM. 
+GraalVM Native Image supports JFR events and users can use [`jdk.jfr.Event`](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.jfr/jdk/jfr/Event.html) API with a similar experience to using JFR in the Java HotSpot VM.
+
+To record JFR events when running a native executable, JFR support and JFR recording must be enabled, and this guide covers how to do that.
+
+> Note: JFR events recording is not supported on GraalVM JDK for Windows. JFR is only supported with native executables built on GraalVM JDK 11.
+
+## Preparation
+
+1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, set the GraalVM home directory as the `$JAVA_HOME` environment variable and add `$JAVA_HOME/bin` to the `PATH` environment variable.
+
```

Comment:
Neither of these might be true. For example, I would expect many users to install GraalVM in a non-user location.

Reply:
I don't think these lines (14-22) are required. We can assume the reader understands the instructions without further details.